src/Viewer.tsx

```diff
@@ -206,6 +213,19 @@ function Viewer(): ReactElement {
 
   // EVENT LISTENERS ////////////////////////////////////////////////////////
 
+  useEffect(() => {
+    const unsubscribe = useViewerStateStore.subscribe(
+      (state) => [state.pendingFrame],
+      ([pendingFrame]) => {
+        if (isTimeSliderDraggedDuringPlayback) {
+          return;
+        }
+        setFrameInput(pendingFrame);
+      }
+    );
+    return unsubscribe;
```

Comment:
The power of pair programming!

```suggestion
    // When user is controlling time slider, do not sync frame input w/ playback
    if (!isTimeSliderDraggedDuringPlayback) {
      return useViewerStateStore.subscribe((state) => state.pendingFrame, setFrameInput);
    }
```

Reply:
This will likely still incur an unnecessary re-render, but if a better data flow exists we can work that out in the likely future PR which moves the time slider into its own component.

Reply:
Opened as https://github.com/allen-cell-animated/timelapse-colorizer/issues/589!